srcpkgs/singularity/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'singularity'
+pkgname=singularity
+version=3.2.1
+revision=1
+archs="~arm* ~aarch*"
```

Comment:
This is not same as `nocross=`. This prevents building on aarch for aarch.

Reply:
I did not think about that. I will finally set `nocross` as it's hard to fix cross compilation and most likely not worth it. 

Thanks